task_02.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A snoozing clock."""
+
+MYDAY = raw_input('What day is it? \nEnter the day as Mon, Tue,' +
+                  ' Wed, Thu, Fri, Sat, or Sun. \n--->>>  ')
+MYTIME = raw_input('What time is it? \ne.g. Enter 8:15 AM as 0815 or 5:43 PM' +
+                   ' as 1743.\n--->>>  ')
+
+SNOOZE = True if (MYDAY == 'Sat' or MYDAY == 'Sun' or MYTIME < '0600') \
```

Comment:
The conditions already return boolean values so you don't need to do `True if ... else False` since they already return those values.

By using a string value for the time your condition is off.